tests/test_decorators.py

```diff
@@ -147,6 +147,50 @@ def f(G):
         for edge in edges:
             self.assertIn(edge, edgelist)
 
+    def test_edgelist(self):
+        @graph_argument('G')
+        def f(G):
+            return G
+
+        edgelist = [(0, 1), (2, 3), (0, 2)]
+
+        nodes, edges = f(edgelist)
+
+        for edge in edges:
+            self.assertIn(edge, edgelist)
```

Comment:
Shouldn't you also verify that when using simply the `edgelist` that the nodes are generated correctly as well?

Reply:
Oops, good catch! should be `self.assertIn(edge, edges)`

Reply:
Yes probably, will add